src/V3FileLine.h

```diff
@@ -265,7 +265,7 @@ class FileLine final {
     }
     void warnOff(V3ErrorCode code, bool flag) { warnOn(code, !flag); }
     bool warnOff(const string& msg, bool flag);  // Returns 1 if ok
-    bool warnIsOff(V3ErrorCode code) const;
+    bool warnIsOff(V3ErrorCode code) const VL_MT_SAFE;
```

Comment:
Also add VL_MT_SAFE to the implementation of this function.   I think then you need more VL_MT_SAFE's added for what it calls.

Reply:
Implementation already have ``VL_MT_SAFE``: https://github.com/verilator/verilator/blob/master/src/V3FileLine.cpp#L372. I've also double-checked that all functions used by implementation are also marked as ``VL_MT_SAFE``